src/types.ts

```diff
@@ -164,6 +167,22 @@ export interface Service {
 	working_dir?: string;
 }
 
+export interface VolumeRef {
```

Comment:
I think it's best we move this into `compose.ts` so that it's clear the module is not actually exporting such a type as it's only needed internally.